src/wallet/api/wallet2_api.h

```diff
@@ -763,6 +763,19 @@ struct WalletManager
     virtual Wallet * openWallet(const std::string &path, const std::string &password, bool testnet = false) = 0;
 
     /*!
+     * \brief  recovers existing wallet using memo (electrum seed)
+     * \param  path           Name of wallet file to be created
+     * \param  password       Password of wallet file
+     * \param  memo           memo (25 words electrum seed)
+     * \param  testnet        testnet
+     * \param  restoreHeight  restore from start height
+     * \return                Wallet instance (Wallet::status() needs to be called to check if recovered successfully)
+     */
+    virtual Wallet * recoveryWallet(const std::string &path, const std::string &password, const std::string &memo,
```

Comment:
Did you mean "recover" ?

Reply:
Ah, I see there are existing functions called recovery, so this is probably to match them. I'm fine either way.

Reply:
yes - i agree more with your suggestions - but they were there already and i didn't want to change everything.

Reply:
You mean the memo too ? That's just confusing, just make it mnemonic. I'd even support changing memo to mnemonic in existing ones, as that one's far from obvious.

Reply:
i would leave the recovery* as those are existing API methods, but change the memo parm to mnemonic.